qst/src/main/java/io/deephaven/qst/array/PrimitiveArrayHelper.java

```diff
@@ -0,0 +1,57 @@
+package io.deephaven.qst.array;
+
+import java.lang.reflect.Array;
+import java.util.Objects;
+
+abstract class PrimitiveArrayHelper<T> {
```

Comment:
I really hate seeing reflection used when we just don't need it. I would revert this and just fix the missing primitive array copies.

Reply:
It's nice to have the growing logic be similar. I've gotten rid of the generic reflection stuff for the arrays, and pushed that logic back to the classes. That seem ok?